rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Great job, here we need both setters and getters for `@book `and `@person `instance variables. Kindly implement that